djangocms_navigation/views.py

```diff
@@ -64,22 +64,23 @@ def get_data(self):
         if pk:
```

Comment:
I didnt want to refactor too heavily here, but I did wonder if this could be simplified further by removing other logic such as this, as it does not appear to be used.